<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TermometroV</class>
 <widget class="QMainWindow" name="TermometroV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>355</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>OpenHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>TermometroV</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../IMAGENES/Xd.png</normaloff>../IMAGENES/Xd.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="grpCent">
      <property name="title">
       <string>Cent√≠grados centigrados</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLCDNumber" name="lcdCent">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialCent">
         <property name="cursor">
          <cursorShape>ClosedHandCursor</cursorShape>
         </property>
         <property name="minimum">
          <number>-273</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpFahr">
      <property name="title">
       <string>Fahrenheit centigrados</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="lcdFahr">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialFahr">
         <property name="cursor">
          <cursorShape>ClosedHandCursor</cursorShape>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Kelvin centigrados</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLCDNumber" name="lcdKelvin">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(144, 144, 144);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialKelvin">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialKelvin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdKelvin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
